olive/passes/pytorch/trt_utils.py

```diff
@@ -0,0 +1,67 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# --------------------------------------------------------------------------
+try:
+    import torch_tensorrt  # noqa: F401
+except ImportError:
+    raise ImportError("Please install torch_tensorrt with: pip install torch-tensorrt")
+
+import io
+import logging
+from contextlib import redirect_stdout
+
+import tensorrt as trt
+import torch
+import torch_tensorrt.fx.tracer.acc_tracer.acc_tracer as acc_tracer
+from torch_tensorrt.fx import InputTensorSpec, TRTInterpreter, TRTModule, compile
+
+
+class TRTLinearLayer(TRTModule):
+    def forward(self, input):
+        """Forward pass of the module. Casts input to fp16 and casts output back to original data type."""
+        return super().forward(input.to(torch.float16)).to(input.dtype)
+
+
+def compile_trt_model(torch_module: torch.nn.Module, hidden_states: torch.Tensor, batch_size: int, seqlen: int):
+    """Compile a torch module to a TensorRT module.
+
+    :param torch_module: The torch module to compile. Only torch.nn.Linear modules are supported currently.
+    :param hidden_states: The input tensor to the torch module.
+    :param batch_size: The batch size of the input tensor.
+    :param seqlen: The maximum sequence length of the input tensor. seqlen dimension is treated as dynamic.
+    """
+
+    logging.getLogger("torch_tensorrt").setLevel(logging.ERROR)
```

Comment:
what's the consideration that we need set the logger level of torch_tensor? would you add some comments here?

Reply:
Yes, I can add comments. 
the torch tensorflow logs are very verbose so the olive logs get over populated. 
It produces many lines per module and there are around or so module per model.

Reply:
Done!